dockerfiles/igv/makeigvpesr_trio.py

```diff
@@ -44,9 +44,9 @@ def ped_info_readin(ped_file):
         pin = line.strip().split()
         if not pin[1] in out.keys():
             out[pin[1]] = [pin[1]]
-        if not(pin[2]) == 0:
+        if not (pin[2]) == 0:
```

Comment:
Same deal as in makeigvpesr_cram:
```suggestion
        if pin[2] != "0":
```